config/ldp/authorization/authorizers/accessCheckers/agent.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Naming convention: `accessCheckers` should become `access-checkers`.

Reply:
Fixed